dataeng/jobs/analytics/StitchSnowflakeLagMonitor.groovy

```diff
@@ -0,0 +1,63 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+
+
+
+class StitchSnowflakeLagMonitor {
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("stitch-snowflake-lag-monitor") {
```

Comment:
Does the `SCHEMA` need to be added to the name of this job - in order to support other jobs monitoring other schemas?

Reply:
Ah, as in an "environment" for each schema?   That seems not unreasonable.

Reply:
I had not expected this to be a full implementation since we were trying to solve a production issue, but since it's already taken too long to be useful for that I'll update everything to make it actually reusable.